cohorts/load.py

```diff
@@ -239,6 +245,7 @@ class Cohort(Collection):
     def __init__(self,
                  patients,
                  cache_dir,
+                 ensembl_version,
```

Comment:
Since Kallisto isn't required when creating a `Cohort`, can we default this to something? Maybe default it to `None` and then raise a `ValueError` in `load_kallisto` if the arg is `None`?